aspnet/RichTextEditor/Working-with-Tables.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: Working with Tables opreation | RTE | Syncfusion | ASP.NET
-description: Working with Tables customization for RichTextEditor control
+title: Working with Tables in ASP.NET Web Forms RichTextEditor | Syncfusion
+description: Learn here all about working with Tables in Syncfusion ASP.NET Web Forms RichTextEditor control, its elements, and more.
```

Comment:
Learn here all about working with tables in Syncfusion ASP.NET Web Forms RichTextEditor control, its elements, and more.